logstream.go

```diff
@@ -69,15 +78,15 @@ func (s *LogStream) findStream() (*cloudwatchlogs.LogStream, error) {
 		return nil, err
 	}
 
-	if len(resp.LogStreams) == 0 {
-		return nil, nil
-	}
+	streams := Filter(resp.LogStreams, func(st *cloudwatchlogs.LogStream) bool {
```

Comment:
`Filter` is great for the general case, but maybe an `any` method makes more sense for this case?

```
func Filter(vs []*cloudwatchlogs.LogStream, f func(*cloudwatchlogs.LogStream) bool) []*cloudwatchlogs.LogStream {
    for _, v := range vs {
        if f(v) {
            return true
        }
    }
    return false
}
```